modules/data.atmosphere/README.md

```diff
@@ -32,7 +32,7 @@ The PEcAn.data.atmosphere package is 'standalone'.
 
 ## PEcAn variable names
 
-See https://pecanproject.github.io/pecan-documentation/latest/met-data.html
+For the most updated list, see https://pecanproject.github.io/pecan-documentation/develop/time-standard.html#input-standards
```

Comment:
I think links in most other docs point to the main branch (confusingly `latest` on github.io and `main` on github.com). Is it important for this one to be the develop version?

Reply:
fixed, I'm not 100% if I understand correctly, but only `master` and `develop` and other branch names in the URL point to an existing page in the documentation.